examples/zephyr/golioth_basics/README.md

```diff
@@ -4,3 +4,146 @@ This example will connect to Golioth and demonstrate Logging, Over-the-Air (OTA)
 firmware updates, and sending data to both LightDB state and LightDB Stream.
 Please see the comments in main.c for a thorough explanation of the expected
 behavior.
+
+## Requirements
+
+* Golioth credentials
+* Network connectivity
+
+## Building and Running
+
+### Authentication specific configuration
+
+#### PSK based auth
```

Comment:
Depending on who merges first, we may need to update this to include information about runtime PSK settings.

Reply:
This sample is set up to show how to use the config struct so it directly passes the kconfig symbols in main.c. We probably need to update the readme, the main.c, or both to match the approach taken in this example.